bedrock/settings/base.py

```diff
@@ -2325,7 +2325,7 @@ def lazy_wagtail_langs():
 WAGTAIL_CONTENT_LANGUAGES = lazy(lazy_wagtail_langs, list)()
 
 # Don't automatically make a page for a non-default locale availble in the default locale
-WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE = False
+WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE = False  # note that WAGTAILLOCALIZE is correct without the _
```

Comment:
Why does this not use the `_` but the one below does? Separate libraries?

Reply:
Yeah, exactly that. `wagtail-localize` uses `WAGTAILLOCALIZE_FOO` (and my eyes cross when seeing those `LL` together), while `wagtail-localize-smartling` uses `WAGTAIL_LOCALIZE_SMARTLING`